src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -211,11 +214,15 @@ int ACA_Dataplane_OVS::update_port_state_workitem(const PortState current_PortSt
     break;
 
   case OperationType::NEIGHBOR_CREATE_UPDATE:
-
+    /* deprecating */
```

Comment:
If we can use some annotation in the codes, that would look more formal :-)

One example in Java: https://github.com/futurewei-cloud/alcor/blob/0276c273e91de79e6b603687810afa730e5a4117/lib/src/main/java/com/futurewei/alcor/common/repo/ICache.java#L23

Reply:
yes, that's a good suggestion, unfortunately the c++ [[deprecated]] keyword is for function level only.